src/components/Toolbar/Toolbar.module.css

```diff
@@ -0,0 +1,30 @@
+.toolbar {
+    display: flex;
+    gap: 10px;
+    align-items: center;
+    padding: var(--spacers-dp8) var(--spacers-dp8) var(--spacers-dp8)
```

Comment:
This might need logical rules, like `padding-block: var(--spacers-dp8)` and `padding-inline: var(--spacers-dp16) var(--spacers-dp8)`